service/history/mutableStateBuilder.go

```diff
@@ -260,22 +260,20 @@ func newMutableStateBuilderWithVersionHistories(
 	return s
 }
 
-func (e *mutableStateBuilder) CopyToPersistence() *persistencespb.WorkflowMutableState {
-	state := &persistencespb.WorkflowMutableState{}
-
-	state.ActivityInfos = e.pendingActivityInfoIDs
-	state.TimerInfos = e.pendingTimerInfoIDs
-	state.ChildExecutionInfos = e.pendingChildExecutionInfoIDs
-	state.RequestCancelInfos = e.pendingRequestCancelInfoIDs
-	state.SignalInfos = e.pendingSignalInfoIDs
-	state.SignalRequestedIds = convertStringSetToSlice(e.pendingSignalRequestedIDs)
-	state.ExecutionInfo = e.executionInfo
-	state.ExecutionState = e.executionState
-	state.NextEventId = e.nextEventID
-	state.BufferedEvents = e.bufferedEvents
-	state.Checksum = e.checksum
-
-	return state
+func (e *mutableStateBuilder) CopyToProto() *persistencespb.WorkflowMutableState {
```

Comment:
Why is it called copy?  This is more like `MutableAsProto`

Reply:
By the way, should we actually considering creating a copy (clone)?

Reply:
Agree, word "copy" might be confusing. Renamed it to `ToProto`. Actual copy is not needed there, as this data get serialized right away (=copy).